dashboard/test/controllers/scripts_controller_test.rb

```diff
@@ -8,6 +8,23 @@ class ScriptsControllerTest < ActionController::TestCase
     @not_admin = create(:user)
   end
 
+  # Some tests in scripts controller assume we're working within the 'levelbuilder' environment.
+  # Run 'set_levelbuiler_env' for a scripts controller test that requires levelbuilder functionality.
+  def set_levelbuilder_env
```

Comment:
do we do this stuff in other tests (like the LevelsControllerTest)? If so it may be worth just putting it in test_helper so it's not duplicated